config/debian/dpkg_client/debian/rules

```diff
@@ -43,10 +28,11 @@ override_dh_strip:
 	# Necessary otherwise the PyInstaller build executables will not run
 
 override_dh_makeshlibs:
-	# Prevent useless call to ldconfig
-	dh_makeshlibs -n
+	# These add a lot of unnecessary dependencies to the deb package because
+	# it just looks for .so files and adds them as dependencies. Since we
+	# are shipping our .so's with Pyinstaller this is unnecessary
 
 override_dh_shlibdeps:
-	# Point debhelper to the locally included shared libraries.
-	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info -l %(ClientBuilder.target_dir)
-
+	# These add a lot of unnecessary dependencies to the deb package because
+	# it just looks for .so files and adds them as dependencies. Since we
```

Comment:
This was added by Sean to make RHEL work. I don't really know what it does, maybe ask him before removing?